### ISSUE TITLE
Files with spaces in their paths are not escaped properly if they are renamed but not modified

### ISSUE THREAD

**Issue by stevethomp** [danger/danger-js#674]
In my Xcode project I moved a lot of files around into new groups, and noticed that some file paths were not being returned correctly from `danger.git.modified_files`. When a file was renamed but not modified, it appeared in the modified files without properly escaping the space in the root folder name, only with the last path component. ie:

```
  'Project/Extensions/UIKit/UIAlertController+Convenience.swift', // Renamed not modified
  'My Project/Extensions/UIKit/UIBarButtonItem+Appearance.swift', // Renamed & modified
```

As far as I can tell I'm doing nothing to change them myself, at the very top of my dangerfile I did a `console.log(danger.git.modified_files);` and that's what was returned.

I'm not sure if this is related to issue #481, but it feels similar. This was an issue for me since I'm calling `swiftlint --path x` on each modified file, but that was failing in the case of a renamed but not modified file.

**Comment by stevethomp** [comment]
Just to update, I traced this back and it looks to me like an issue in the `parse-diff` library. I submitted a PR that I think would fix it to them, https://github.com/sergeyt/parse-diff/pull/22.

**Comment by orta** [comment]
kudos!

**Comment by stevethomp** [comment]
Closing since this was fixed in `parse-diff` `0.5.1`, and Danger was updated to pull in that latest version. Thanks!!!

**Comment by orta** [comment]
Thanks!
